docs/introduction/tutorials_intro.md

```diff
@@ -110,145 +114,158 @@ And the following tutorial:
 
 ## Power Platform APIs
 
-| API | Examples |
+| API | Snippets |
 | --- | --- |
-| DeepSleepLock | [DeepSleepLock example](../apis/power-apis.html#example) |
-| LowPowerTicker | [LowPowerTicker example](../apis/lowpowerticker.html#lowpowerticker-example) | |
-| LowPowerTimeout | [LowPowerTimeout example](../apis/lowpowertimeout.html#lowpowertimeout-example) | |
-| LowPowerTimer | [LowPowerTimer example](../apis/lowpowertimer.html#lowpowertimer-example) | |
-| PowerManagement | [PowerManagement example](../apis/power-management-sleep.html#example) |
+| DeepSleepLock | [DeepSleepLock snippet](../apis/power-apis.html#example) |
+| LowPowerTicker | [LowPowerTicker snippet](../apis/lowpowerticker.html#lowpowerticker-example) | |
+| LowPowerTimeout | [LowPowerTimeout snippet](../apis/lowpowertimeout.html#lowpowertimeout-example) | |
+| LowPowerTimer | [LowPowerTimer snippet](../apis/lowpowertimer.html#lowpowertimer-example) | |
+| PowerManagement | [PowerManagement snippet](../apis/power-management-sleep.html#example) |
 
 ## Memory Platform APIs
 
-| API | Examples |
-| --- | --- |
-| Memory tracing (mbed_mem_trace) | [Memory tracing example](../apis/memory-apis.html#memory-tracing-example) |
-| MPU management (mpug_mgmt) | [MPU management example](../apis/mpu-management.html#example) |
-| MemoryPool | [MemoryPool example](../apis/memorypool.html#memorypool-example) | |
-| Mbed statistics  (mbed_stats)| [Memory statistics example](../apis/mbed-statistics.html#memory-statistics-example) <br> [Thread statistics example](../apis/mbed-statistics.html#thread-statistics-example) <br> [System information example](../apis/mbed-statistics.html#system-information-example) <br> [CPU statistics example](../apis/mbed-statistics.html#cpu-statistics-example)|
+| API | Snippets | Tutorials | Official examples |
+| --- | --- | --- | --- |
+| Memory tracing (mbed_mem_trace) | [Memory tracing snippet](../apis/memory-apis.html#memory-tracing-example) | [Runtime memory tracing](../apis/runtime-memory-tracing.html) | |
+| MPU management (mpug_mgmt) | [MPU management snippet](../apis/mpu-management.html#example) | | |
+| MemoryPool | [MemoryPool snippet](../apis/memorypool.html#memorypool-example) | | |
+| Mbed statistics  (mbed_stats)| - [Memory statistics snippet](../apis/mbed-statistics.html#memory-statistics-example) <br> - [Thread statistics snippet](../apis/mbed-statistics.html#thread-statistics-example) <br> - [System information snippet](../apis/mbed-statistics.html#system-information-example) <br> - [CPU statistics snippet](../apis/mbed-statistics.html#cpu-statistics-example)|  | - [CPU statistics official example](https://github.com/ARMmbed/mbed-os-example-cpu-stats) </br> - [CPU usage official example](https://github.com/ARMmbed/mbed-os-example-cpu-usage) </br> - [Thread statistics official example](https://github.com/ARMmbed/mbed-os-example-thread-statistics)|
```

Comment:
The column title already says that the example if official so it does not need to be repeated